corehq/sql_db/util.py

```diff
@@ -150,3 +153,33 @@ def _get_all_nested_subclasses(cls):
         if subclass not in seen:
             seen.add(subclass)
             yield subclass
+
+
+STANDBY_DELAY_FRESHNESS = 30
+
+
+@quickcache(timeout=STANDBY_DELAY_FRESHNESS)
```

Comment:
this should be `@quickcache(['db_alias'], timeout=STANDBY_DELAY_FRESHNESS)`

Reply:
Yes, updated!